ikarus/controlroutines/loadcontrol.hh

```diff
@@ -74,7 +74,10 @@ public:
         loadSteps_{loadSteps},
         parameterBegin_{tbeginEnd[0]},
         parameterEnd_{tbeginEnd[1]},
-        stepSize_{(parameterEnd_ - parameterBegin_) / loadSteps_} {}
+        stepSize_{(parameterEnd_ - parameterBegin_) / loadSteps_} {
+    if (loadSteps_ <= 0)
```

Comment:
I didn't change `loadSteps_` to `std::size_t` because for undeformed (or initial) state, we use `loadSteps_ = -1`.